rpcs3/Emu/System.cpp

```diff
@@ -1659,38 +1659,14 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 
 			m_force_boot = false;
 		}
-		else if (IsPaused())
```

Comment:
I'm not sure why this code exists, but I think you can use the following change in the part above:
```
ensure(IsReady());
if (m_force_boot || g_cfg.misc.autostart)
```

Reply:
Done.